nailgun/entities.py

```diff
@@ -4242,6 +4242,7 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         ignore.add('root_pass')
         ignore.add('included')
         ignore.add('excluded')
+        ignore.add('token')
```

Comment:
don't you need some condition here, so you can read it in your test?

Reply:
This may be necessary, but at this point there are no tests trying to access this token parameter.